cohorts/load.py

```diff
@@ -215,10 +223,16 @@ def __init__(self,
                  join_how="inner",
                  check_provenance=False,
                  polyphen_dump_path=None,
-                 pageant_coverage_path=None):
+                 pageant_coverage_path=None,
+                 variant_type="snv",
```

Comment:
Yea I think I prefer the alternative, but also see the issue that you can pass one to `load_variants` and  a different set to `load_effects` which could be confusing, so I'm fine with this